backend/polygon/polygon_api.py

```diff
@@ -0,0 +1,19 @@
+"""
+Polygon API class to get stock financials and news articles
+https://polygon.io/docs/stocks/
+https://polygon-api-client.readthedocs.io/en/latest/index.html
+"""
+from polygon import RESTClient # type: ignore
+import config #contains API_KEY
+
+class PolygonAPI():
+    def __init__(self) -> None:
+        self.client = RESTClient(api_key=config.API_KEY) # type: ignore
+
+    def get_financials(self, ticker: str) -> object:
```

Comment:
Specify the object type - ideally would be good to create our own class (so that we can confine the polyon import to this file), or at a minimum define it using the library types.